pnpm-lock.yaml

```diff
@@ -1,4 +1,4 @@
-lockfileVersion: '6.1'
+lockfileVersion: '6.0'
```

Comment:
These changes shouldn't be here 🤔 
Looks like my main may have been out of sync, I'll remove!